pkgs/applications/science/biology/jellyfish/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+    pname = "jellyfish";
+    version = "2.3.0";
+
+    src = fetchurl {
+        url = "https://github.com/gmarcais/Jellyfish/releases/download/v${version}/${pname}-${version}.tar.gz";
+        sha256 = "e195b7cf7ba42a90e5e112c0ed27894cd7ac864476dc5fb45ab169f5b930ea5a";
+    };
+
```

Comment:
Do we need to do something else? Seems to be a little to little to do.

Reply:
@SuperSandro2000 It seems Nix automatically runs `configure` and `make`? This is all I need to do on Linux to get it built and symlinked :thinking: